package.json

```diff
@@ -67,6 +67,7 @@
     "@scure/base": "^1.1.3",
     "@types/debug": "^4.1.7",
     "debug": "^4.3.4",
+    "lodash": "^4.17.21",
```

Comment:
Should we consider adding the single [`lodash.memoize`](https://www.npmjs.com/package/lodash.memoize) package instead?

Reply:
Seems like a good compromise for now. I'm not a fan of Lodash in general, but the best other alternative ([`memoize`](https://github.com/sindresorhus/memoize)) is ESM-only so we can't use that at the moment, and I don't think it's worth the effort to write a proper memoize function either.